packages/legend-extension-dsl-persistence/src/graph/metamodel/pure/model/packageableElements/persistence/DSL_Persistence_ActionIndicatorFields.ts

```diff
@@ -32,9 +33,11 @@ export abstract class DeleteIndicator extends ActionIndicatorFields {
 }
 
 export class DeleteIndicatorForGraphFetch extends DeleteIndicator {
+  deleteFieldPath!: object;
```

Comment:
same comment, please annotate this discrepancy

Reply:
Updated.